amqp_influxdb/__init__.py

```diff
@@ -39,10 +42,31 @@ def __init__(self,
                  exchange,
                  routing_key,
                  message_processor,
-                 connection_parameters=None):
```

Comment:
This should remain `connection_parameters=None` 
and `connection_parameters = connection_parameters or {}`
because `{}` is mutable and remains between invocations